html/welcome.html

```diff
@@ -83,21 +83,21 @@
 
     <div class="container">
 
-        <table class="center" width="768px">
```

Comment:
So that width was simply redundant.

Reply:
Found it in https://github.com/bgruening/docker-galaxy-stable#galaxys-config-settings--toc

Reply:
Apparently yes, since it was set at the CSS level for "center" with the same value. A different value would have overriden the CSS "center" one for the table.